src/qt/paymentserver.h

```diff
@@ -77,7 +77,7 @@ class PaymentServer : public QObject
     // to read from the file specified in the -rootcertificates setting,
     // or, if that's not set, to use the system default root certificates.
     // If you pass in a store, you should not X509_STORE_free it: it will be
-    // freed either at exit or when another set of CAs are loaded.
+    // freed either at exit or when another set of CA:s are loaded.
```

Comment:
why?

Reply:
Yeah I would add these to `test/lint/lint-spelling.ignore-words.txt` rather than use `:` to separate them.